plasmapy/plasma/tests/test_equilibria1d.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+from astropy import units as u
+
+from plasmapy.formulary import magnetic_pressure
+from plasmapy.plasma.equilibria1d import HarrisSheet
+
+
+def test_HarrisSheet():
+    B0 = 1 * u.T
+    delta = 1 * u.m
+    P0 = 0 * u.Pa
+    hs = HarrisSheet(B0, delta, P0)
+    B = hs.magnetic_field(0 * u.m)
+    assert u.isclose(
+        B, 0 * u.T, atol=1e-9 * u.T
+    ), "Magnetic field is supposed to be zero at Y=0"
+
+
+def test_pressure_balance():
+    B0 = 1 * u.T
+    delta = 1 * u.m
+    P0 = 0 * u.Pa
+    hs = HarrisSheet(B0, delta, P0)
+    y = [-7, -3, 0, 2, 47] * u.m
+    B = hs.magnetic_field(y)
+    P = hs.plasma_pressure(y)
+    p_b = magnetic_pressure(B)
+    total_pressure = P + p_b
+    assert u.allclose(total_pressure, total_pressure[0], atol=1e-9 * u.Pa)
+
+
+def test_currentDensity():
+    B0 = 1 * u.T
+    delta = 1 * u.m
+    P0 = 0 * u.Pa
+    hs = HarrisSheet(B0, delta, P0)
+    y = [-2, 0, 2] * u.m
+    J = hs.current_density(y)
+    correct_J = [56250.656, 796178.343, 56250.656] * u.A / u.m**2
```

Comment:
It looks like there are small differences in the current densities too.  
```E        +  where False = <function allclose at 0x7fe7e7b114e0>(<Quantity [ 56222.1400139 , 795774.71502628,  56222.1400139 ] A / m2>, <Quantity [ 56250.656, 796178.343,  56250.656] A / m2>, atol=((1e-09 * Unit("A")) / (Unit("m") ** 2)))
```
It looks like there weren't any problems with unit conversions like I thought.